### DISCUSSION TITLE
Rendering for absorption

### DISCUSSION THREAD

**Discussion by Pietersielie** [idia-astro/iDaVIE#403] [Medium-term features]
At the moment, the rendering shader (a ray-marching algorithm) samples values along the ray and returns either the maximum or the average value, which provides a suitable approximation to emission. To account for absorption in the foreground, a new shader will have to be developed, possibly based on [radiative transfer](https://en.wikipedia.org/wiki/Radiative_transfer) equations.

**Comment by paoloserra** [comment]
@Pietersielie a while back we discussed a possible solution for emission and absorption. The main idea is captured at https://github.com/idia-astro/iDaVIE/issues/256 .
